node/src/client.rs

```diff
@@ -278,7 +275,7 @@ impl sc_client_api::BlockBackend<Block> for Client {
 		match_client!(self, block_status(id))
 	}
 
-	fn justifications(&self, id: &BlockId<Block>) -> sp_blockchain::Result<Option<Justifications>> {
+	fn justifications(&self, id: H256) -> sp_blockchain::Result<Option<Justifications>> {
```

Comment:
explicit type should not be used here, i think it should be `<Block as BlockT>::Hash`